pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -219,6 +225,73 @@ func Convert_v1beta1_RollingUpdateDeployment_To_extensions_RollingUpdateDeployme
 	return nil
 }
 
+func Convert_extensions_DaemonSetSpec_To_v1beta1_DaemonSetSpec(in *extensions.DaemonSetSpec, out *DaemonSetSpec, s conversion.Scope) error {
```

Comment:
Hmm, I had some errors with validation earlier and thought that it will help. It turns out that only
`Convert_extensions_RollingUpdateDaemonSet_To_v1beta1_RollingUpdateDaemonSet                 Convert_v1beta1_RollingUpdateDaemonSet_To_extensions_RollingUpdateDaemonSet,`
are required